features/upgrade/workloads/descheduler-upgrade.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Descheduler major upgrade should work fine
+  # @author knarra@redhat.com
+  @upgrade-prepare
+  @admin
+  @destructive
+  Scenario: Upgrading descheduler from a major to major version should work fine - prepare
+    Given I store master major version in the clipboard
+    Given I switch to cluster admin pseudo user
+    Given kubedescheduler operator has been installed successfully
+    Given I use the "openshift-kube-descheduler-operator" project
+    Given I obtain test data file "descheduler/kubedescheduler-<%= cb.master_version %>.yaml"
+    When I run the :create admin command with:
+      | f | kubedescheduler-<%= cb.master_version %>.yaml |
+    Then the step should succeed
+    Given I wait up to 120 seconds for the steps to pass:
+    """
+    And status becomes :running of exactly 1 pods labeled:
+      | app=descheduler |
+    """
+    Given evaluation of `pod.name` is stored in the :pod_name clipboard
+    Given I wait up to 180 seconds for the steps to pass:
+    """
+    When I run the :logs client command with:
+      | resource_name | pod/<%= cb.pod_name %> |
```

Comment:
@liangxia since this will be a fresh install of descheduler it does not generate very long log, also descheduler interval seconds is set to 3600, so there will not be any further log generate until that time. So i feel above should be okay.